apps/isitchristmas/is_it_christmas.star

```diff
@@ -50,7 +50,7 @@ def main(config):
     day = now.day
     month = now.month
 
-    christmas_date = config.get("christmas_date")
+    christmas_date = config.get("christmas_date", "12-25")
```

Comment:
Im not sure how Tidbyt handles the dates for international users, but this kind of makes me feel like we shouldnt be hardcoding a date here?

For example, a user in Russia might celebrate Christmas on January 7 - so this hardcoded date would be incorrect.

Maybe I am misunderstanding how dates work here?

Reply:
There is a config for different dates. This is just a default value.